lib/google/index.js

```diff
@@ -87,3 +96,43 @@ exports.verifyPayment = function (payment, cb) {
 		});
 	});
 };
+
+
+exports.cancelSubscription = function (payment, cb) {
+	var keyObject;
+
+	try {
+		keyObject = validatePaymentAndParseKeyObject(payment);
+	} catch (error) {
+		return process.nextTick(function () {
+			cb(error);
+		});
+	}
+
+	/* jshint camelcase:false */
```

Comment:
We use ESLint now, so this probably doesn't do much

Reply:
removed